src/ui/pages/AddEthereumChain/RpcUrlForm/RpcUrlForm.tsx

```diff
@@ -46,24 +51,34 @@ const Field = React.forwardRef(
 
 export function RpcUrlForm({
   network,
-  oldNetwork,
+  prevNetwork,
   isSubmitting,
   onSubmit,
   onCancel,
   rpcUrlHelpHref,
 }: {
   network: NetworkConfig;
-  oldNetwork: NetworkConfig;
+  prevNetwork: NetworkConfig;
   rpcUrlHelpHref: string;
   isSubmitting: boolean;
   onSubmit: (result: NetworkConfig) => void;
   onCancel: () => void;
 }) {
+  const { networks } = useNetworks();
+
+  const currentRpcUrl = networks?.getRpcUrlInternal(
```

Comment:
`Networks.getNetworkRpcUrlInternal(prevNetwork);`?

Reply:
As I can see `getRpcUrlInternal` has an extra check for existing network